core/Pos/Core/Timestamp.hs

```diff
@@ -53,3 +56,6 @@ diffTimestamp t1 t2 = getTimestamp t1 - getTimestamp t2
 
 addMicrosecondsToTimestamp :: Microsecond -> Timestamp -> Timestamp
 addMicrosecondsToTimestamp m t = Timestamp { getTimestamp = (getTimestamp t) + m }
+
+timestampToPosix :: Timestamp -> POSIXTime
+timestampToPosix (Timestamp ts) = fromIntegral ts / 1000000
```

Comment:
As far as I understand, in `cardano-sl-0.5` there is no explorer in this project yet, but I can replace that function on merging this PR to `master`